demeter/utils/image_3d_visualisation.py

```diff
@@ -350,8 +350,10 @@ def update(val):
 
 from warnings import simplefilter
 simplefilter(action='ignore', category=DeprecationWarning)
-vedo.embedWindow('ipyvtk')
-vedo.settings.useDepthPeeling = True  # if you use transparency <1
+# XXX comment out vedo lines temporarily to avoid error (maybe vedo update?):
+# AttributeError: module 'vedo' has no attribute 'embedWindow' for some reason, maybe a change in last
+# vedo.embedWindow('ipyvtk')
```

Comment:
This is probably the most controversial change, this may break stuff in this module.

Maybe I did not install `vedo` correctly but for me `vedo.embedWindow` did not exist ...